tests/functional/sources/test_source_fresher_state.py

```diff
@@ -90,7 +90,6 @@ def _assert_freshness_results(self, path, state):
             == "https://schemas.getdbt.com/dbt/sources/v3.json"
         )
         assert data["metadata"]["dbt_version"] == dbt.version.__version__
-        assert data["metadata"]["invocation_id"] == dbt.tracking.active_user.invocation_id
```

Comment:
Unless I'm misunderstanding the nature of this test, the invocation_id shouldn't really matter, and the change in dbt's behavior, which now resets the invocation_id on every invocation, was causing this check to fail. Therefore, I removed it.